docs/assets/js/search.js

```diff
@@ -9,43 +9,56 @@ const searchResultsElm = document.getElementById( 'search-results' );
  * Update page markup with search results.
  * @param {Array} results - A list of search result hits as objects.
  * @param {Object} store - search index/meta data store in the window object.
+ * @returns {boolean} True if there are search results, false otherwise.
```

Comment:
Curious: what's the value in returning a boolean? Is there a future use case for it?

Reply:
It was purely in service of the guard clause that returned the function early. That could also return undefined instead.

Reply:
Refactored away in https://github.com/cfpb/design-system/pull/479/commits/04489eac20472c1a1c293054c09e98dcee1c100b